app/client/src/components/pages/Community.Tabs.Restore.js

```diff
@@ -475,9 +487,29 @@ function Restore() {
                       >
                         {sortedStoriesData.map((item) => (
                           <AccordionItem
-                            ariaLabel={item.ss_title}
+                            ariaLabel={item.ss_title ?? 'Unknown'}
                             key={item.ss_seq}
-                            title={<strong>{item.ss_title}</strong>}
+                            title={
+                              <strong>{item.ss_title ?? 'Unknown'}</strong>
+                            }
+                            subTitle={
+                              <>
+                                <>
+                                  Publication Year:&nbsp;&nbsp;
+                                  {item.type1_fiscal_year ?? 'N/A'}
+                                  <br />
+                                </>
+                                <>
+                                  Pollutants Addressed:&nbsp;&nbsp;
+                                  {item.pollutants ?? 'N/A'}
+                                  <br />
+                                </>
+                                <>
+                                  Sources of Pollution: &nbsp;&nbsp;
+                                  {item.poll_sources ?? 'N/A'}
+                                </>
+                              </>
+                            }
```

Comment:
We can talk about this during the meeting. 

To be consistent with other tabs, I think all of these should be inside the accordion in a table above the story text. It looks like pollutants addressed and sources of pollution could both be comma delimited lists at some point, which could make the subtitle area crowded. 

I think publication year can be both in the subtitle and inside the accordion. Kind of like what we do with organization name for monitoring stations. Maybe we should add sorting as well.